schema/me/update_me_mutation.js

```diff
@@ -0,0 +1,81 @@
+import { GraphQLString, GraphQLInt, GraphQLInputObjectType } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+
+import UserType from "../user"
+
+export const EditableLocationFields = new GraphQLInputObjectType({
+  name: "EditableLocation",
+  fields: {
+    address: {
+      description: "First line of an address",
+      type: GraphQLString,
+    },
+    address_2: {
+      description: "Second line of an address",
+      type: GraphQLString,
+    },
+    city: {
+      description: "The city the location is based in",
+      type: GraphQLString,
+    },
+    country: {
+      description: "The county the location is based in",
+      type: GraphQLString,
+    },
+    summary: {
+      description: "An optional display string for the location",
+      type: GraphQLString,
+    },
+    postal_code: {
+      description: "Postal code for a string",
+      type: GraphQLString,
+    },
+    state: {
+      description: "The (optional) name of the state for location",
+      type: GraphQLString,
+    },
+    state_code: {
+      description: "The (optional) state code of the state for location",
+      type: GraphQLString,
+    },
+  },
+})
+
+export default mutationWithClientMutationId({
+  name: "UpdateMyProfile",
+  description: "Update the current logged in user.",
+  // This is just a subset needed today, anything inside the User object
```

Comment:
Eh, I would leave the comment out. To be specific, it's anything that is settable by the API endpoint: https://github.com/artsy/gravity/blob/67d8c6ed880ca8e0c9771f26861bff783a6e763d/app/api/v1/me_endpoint.rb#L17-L20 , which currently is `collector_level` (actually stored separately from `User`), plus a bunch of `User` fields (coming from `valid_hash_fields`).